### ISSUE TITLE
Trigger plots won't save when on automatic testing mode

### ISSUE THREAD

**Issue by eurunuela** [physiopy/phys2bids#287]
## Expected Behavior

When running pytest on CircleCI, Travis and Azure, the trigger plots should get saved.

## Actual Behavior

Trigger plots don't get saved when running pytest on CircleCI, Travis and Azure. The `plt.savefig()` function gets stuck.

## Specifications

    - Python version: 3.6 and 3.7
    - phys2bids version: v2.1.0+29.g2007d8b
    - Platform: CircleCI, Travis and Azure Pipelines

**Comment by vinferrer** [comment]
I want to update this. Actually since you added the flag to not execute these test in circleCI, I think they do not execute in local anymore neither:
![image](https://user-images.githubusercontent.com/38909338/93306849-ae871c00-f800-11ea-8533-81863d4f455a.png)

**Comment by eurunuela** [comment]
Indeed, I added a condition stating that we expect it to fail and that's exactly what you're seeing. We're gonna have to debug that.
